regparser/tree/xml_parser/appendices.py

```diff
@@ -302,6 +315,12 @@ def is_subhead(tag, text):
                 self.paragraph_with_marker(
                     text,
                     tree_utils.get_node_text_tags_preserved(child))
+            elif child.tag == 'SEQUENCE':
+                old_depth = self.depth
+                self.end_group()
+                self.depth = old_depth
+                # import pdb; pdb.set_trace();
+                self.process_sequence(child)
```

Comment:
I'm not sure I've seen a `SEQUENCE` element. What's it do? Can we have an example in a unit test?